organisations.py doesn't type check

```
mypy ietfdata/*.py ietfdata/tools/*.py
ietfdata/tools/organisations.py:322: error: Statement is unreachable  [unreachable]
Found 1 error in 1 file (checked 12 source files)
```

--- a/ietfdata/tools/organisations.py
+++ b/ietfdata/tools/organisations.py
@@ -318,7 +318,6 @@ class OrganisationMatcher:
 
 
     def add(self, organisation:str, email:str):
-        assert organisation is not None
         if organisation == "":
             warnings.warn("organisation is an empty string, ignored.")
             return

--- a/ietfdata/tools/organisations.py
+++ b/ietfdata/tools/organisations.py
@@ -318,9 +318,7 @@ class OrganisationMatcher:
 
 
     def add(self, organisation:str, email:str):
-        if organisation is None:
-            warnings.warn("organisation is None, ignored.")
-            return 
+        assert organisation is not None
         if organisation == "":
             warnings.warn("organisation is an empty string, ignored.")
             return

